README.md

```diff
@@ -145,6 +145,23 @@ seconds past the minute).
 A big shout out to [rufus-scheduler](http://github.com/jmettraux/rufus-scheduler)
 for handling the heavy lifting of the actual scheduling engine.
 
+### Enabling/Disabling jobs
+
+By default all the jobs are enabled. There is an option at job's definition to change
+this behavior.
+
+``` yaml
+clear_leaderboards_moderator:
+  every: "30s"
+  class: CheckDaemon
+  enabled: false
+  description: "This job is disabled by default"
+```
+
+However, the flag can be changed through the web interface at the Recurring Jobs tab.
+
+NOTE: All the changes made after loading the schedule, for example at the web, overrides
```

Comment:
NOTE: Changes made thorugh the UI supersede job's configuration, even after a restart.